09-paradigms/06-composition/05-exercises/01-hof/README.md

```diff
@@ -52,5 +51,5 @@ incluirán un boilerplate.
 
 ## Recursos
 
-* https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions_and_function_scope
-* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/prototype
+* [Functions_and_function_scope  -MDN](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions_and_function_scope)
```

Comment:
⚠️ Espacios y underscores:

`[Functions_and_function_scope  -MDN]` => [Functions and function scope - MDN]

Reply:
Corregido :+1: